corehq/apps/reports/static/reports/ko/export.manager.js

```diff
@@ -316,6 +316,24 @@ var ExportManager = function (o) {
         });
     };
 
+    self.requestMultimediaDownload = function(data, event){
+        var $button = $(event.srcElement || event.currentTarget),
+            xmlns = $button.data('xmlns'),
+            downloadUrl = $button.data('downloadurl') + '&xmlns=' + xmlns,
+            title = $button.data('modulename');
+
+        title = $button.data('formname').length ? title + " > " + $button.data('formname') : title;
+
+        resetModal("'" + title + "' (multimedia)", true);
+
+        self.downloadExport({
+            downloadUrl: downloadUrl,
+            xmlns: xmlns,
+            isBulkDownload: "multimedia",
```

Comment:
Why is this a string? Looks like this is only interpreted as a boolean. It's still truthy, so it doesn't really matter, just curious.